src/logic/exceptions/CodedException.ts

```diff
@@ -25,27 +25,41 @@ export class CodedException extends Error {
     this.context = context
   }
 
-  /**
-   * Log the error in the console and send to Sentry
-   */
-  public log(isTracked = true): void {
+  public log(): void {
+    // Filter out the logError fn from the stack trace
+    if (this.stack) {
+      const newStack = this.stack
+        .split('\n')
+        .filter((line) => !line.includes(logError.name))
+        .join('\n')
+      try {
+        this.stack = newStack
+      } catch (e) {}
```

Comment:
out of ignorance, what's the purpose of this? Under what condition the assignment may throw?

Reply:
There's a scary warning on [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/Stack) about this property. I suspect in some browsers it might be read only or something.